datafusion/core/tests/dataframe_functions.rs

```diff
@@ -667,14 +667,14 @@ async fn test_fn_substr() -> Result<()> {
 async fn test_cast() -> Result<()> {
     let expr = cast(col("b"), DataType::Float64);
     let expected = vec![
-        "+-------------------------+",
-        "| CAST(test.b AS Float64) |",
-        "+-------------------------+",
-        "| 1                       |",
-        "| 10                      |",
-        "| 10                      |",
-        "| 100                     |",
-        "+-------------------------+",
+        "+--------+",
+        "| test.b |",
```

Comment:
isn't the original header better？
@alamb @andygrove

Reply:
I personally don't think seeing the cast in the column name adds much value. Also no cast in the subject is consistent with postgres:


```sql
alamb=# select cast(1 as int);
 int4 
------
    1
(1 row)

alamb=# select cast(i as int) from foo;
 i 
---
 1
 2
 0
(3 rows)
```